core/templates/dev/head/pages/exploration-player-page/services/extract-image-filenames-from-state.service.ts

```diff
@@ -117,9 +117,9 @@ oppia.factory('ExtractImageFilenamesFromStateService', [
      * @param {string} strHtml - The string from which the object of
      *                           filepath should be extracted.
      */
-    var _extractFilepathValueFromOppiaNonInteractiveImageTag = function(
+    var _extractImageIdValueFromOppiaNonInteractiveImageTag = function(
```

Comment:
Please update the above comments accordingly.

Reply:
done